server/flyway/sql/V30__api_lambda_db_user_restrict_access.sql

```diff
@@ -0,0 +1,38 @@
+
+-- Revoke default privileges previously assigned
+ALTER DEFAULT privileges IN SCHEMA app_fam
+REVOKE SELECT, INSERT, UPDATE, DELETE ON tables FROM ${api_db_username};;
+
+-- Revoke broad permissions previously assigned
+REVOKE SELECT, UPDATE, DELETE, INSERT ON ALL TABLES IN SCHEMA app_fam FROM ${api_db_username};
+
```

Comment:
Just curious, as a learning question, is the line 7 to revoke broad permissions (ON ALL TABLES) already covers the need for line 4?

Reply:
No. Line 4 revokes broad permissions on tables created in the future. Line 7 revokes broad permissions on existing tables.